ros/api/v0/hosts.py

```diff
@@ -222,3 +222,71 @@ def get(self, host_id):
                   .format(host_id))
 
         return record
+
+
+class HostHistoryApi(Resource):
+    display_performance_score_fields = {
+        'cpu_score': fields.Integer,
+        'memory_score': fields.Integer,
+        'io_score': fields.Integer,
+        'report_date':  fields.String
+    }
+
+    data_fields = {
+        'inventory_id': fields.String,
+        'performance_history': fields.List(fields.Nested(display_performance_score_fields))
+    }
+    meta_fields = {
+        'count': fields.Integer,
+        'limit': fields.Integer,
+        'offset': fields.Integer
+    }
+    links_fields = {
+        'first': fields.String,
+        'last': fields.String,
+        'next': fields.String,
+        'previous': fields.String
+    }
+    history_fields = {
+        'meta': fields.Nested(meta_fields),
+        'links': fields.Nested(links_fields),
+        'data': fields.Nested(data_fields)
```

Comment:
We cannot do this in case of API which supports pagination, because `build_paginated_system_list_response` only accept one parameter to construct the output response.